src/sdk-reference/js/6/kuzzle-event-emitter/prependOnceListener/snippets/prepend-once-listener.test.yml

```
@@ -0,0 +1,9 @@
+name: kuzzleEventEmitter#prepend-once-listener
+description: Prepends an once new listener for the given event
+hooks:
+  before:
+  after:
+template: eventemitter
+expected: Successfully prepended 1 new once listener
```

If my previous remark is applied, then this line needs to be changed for the following:

```suggestion
expected: 
  - ^listener2$
  - ^listener1$
  - ^listener1$
```